lib/racecar/ctl.rb

```diff
@@ -36,10 +36,14 @@ def liveness_probe(args)
       require "racecar/liveness_probe"
       parse_options!(args)
 
-      if ENV["RAILS_ENV"]
+      if ENV["RAILS_ENV"] && File.exist?("config/racecar.yml")
         Racecar.config.load_file("config/racecar.yml", ENV["RAILS_ENV"])
```

Comment:
Perhaps we should use [this method](https://github.com/zendesk/racecar/blob/master/lib/racecar/rails_config_file_loader.rb#L4) for loading the config?

Reply:
@dasch I've updated it to behave exactly as the cli load them

Reply:
I am wondering if loading the config like this (`RailsConfigFileLoader`) is the right thing here, doesn't that mean the liveness probe will always try to boot the entire Rails app just to check the modified time of a file?
There is the `without_rails` config option, but what if the consumer needs Rails, then the liveness probe will do all that extra work for nothing.